modules/ossm-enabling-jaeger.adoc

```diff
@@ -22,9 +22,9 @@ spec:
     type: Jaeger
 ----
 
-Currently, the only tracing type that is supported is `Jaeger`.
+In {SMProductName} 2.6, the tracing type `Jaeger` is deprecated and disabled by default.
 
-Jaeger is enabled by default. To disable tracing, set `type` to `None`.
+In {SMProductName} 2.5 and earlier, the tracing type `Jaeger` is enabled by default. To disable it, set the `spec.tracing.type` parameter of the `ServiceMeshControlPlane` resource to `None`.
```

Comment:
```suggestion
In {SMProductName} 2.5 and earlier, the tracing type `Jaeger` is enabled by default. To disable `Jaeger` tracing, set the `spec.tracing.type` parameter of the `ServiceMeshControlPlane` resource to `None`.
```
"it" is never really clear; and if I search for "disable Jaeger," I won't find it if the keywords are "it" instead of "disable Jaeger"
make sense?